clinica/pipelines/dwi_connectome/dwi_connectome_utils.py

```diff
@@ -15,33 +15,59 @@ def get_luts():
     return [default, a2009s]
 
 
+def get_conversion_luts_offline():
+
+    # TODO: use this function if no internet connect found in client (need to upload files to clinica repository)
+    return
+
+
 def get_conversion_luts():
-    import os
+    from os import pardir
+    from os.path import abspath, dirname, join
+    from pathlib import Path
 
-    from clinica.utils.exceptions import ClinicaException
+    from clinica.utils.inputs import RemoteFileStructure, fetch_file
+    from clinica.utils.stream import cprint
 
-    try:
-        # For aparc+aseg.mgz file:
-        default = os.path.join(
-            os.environ["MRTRIX_HOME"],
-            "share",
-            "mrtrix3",
-            "labelconvert",
-            "fs_default.txt",
-        )
-        # For aparc.a2009s+aseg.mgz file:
-        a2009s = os.path.join(
-            os.environ["MRTRIX_HOME"],
-            "share",
-            "mrtrix3",
-            "labelconvert",
-            "fs_a2009s.txt",
-        )
+    root = dirname(abspath(join(abspath(__file__), pardir, pardir)))
 
-        # TODO: Add custom Lausanne2008 conversion LUTs here.
-    except KeyError:
-        raise ClinicaException("Could not find MRTRIX_HOME environment variable.")
-    return [default, a2009s]
+    path_to_mappings = Path(root) / "resources" / "mappings"
+
+    url_mrtrix = "https://raw.githubusercontent.com/MRtrix3/mrtrix3/master/share/mrtrix3/labelconvert/"
+
+    FILE1 = RemoteFileStructure(
```

Comment:
To avoid triggering a `variable should be lowercase` warning.

```suggestion
    fs_default = RemoteFileStructure(
```